README.md

```diff
@@ -29,4 +29,21 @@ $CODE/statesman-server/config/local.yml
 $CODE/statesman-server/docker-compose.yml
 ```
 
-
+To debug the statesman service running via Docker in IntelliJ, follow below mentioned steps:
+```shell
+# Uncomment below mentioned line from $CODE/statesman-server/Dockerfile
+EXPOSE 8084
+
+# Uncomment below mentioned lines from $CODE/statesman-server/docker-compose.yml
+- "8084:8084"
+- JAVA_OPTS: -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8084
```

Comment:
I suggest we should rename JAVA_OPTS to DEBUG_JAVA_OPTS, to call out its usage only for debug

Reply:
Sounds good. I will update the same.

Reply:
Updated